cypress/support/commands.ts

```diff
@@ -62,7 +62,9 @@ class CustomizedBridge extends _Eip1193Bridge {
       return wrapResponse([Cypress.env("PRIVATE_TEST_WALLET_ADDRESS")])
     }
     if (method === "eth_chainId") {
-      return wrapResponse(`0x${Cypress.env("NETWORK_ID").toString(16).toUpperCase()}`)
+      return wrapResponse(
+        `0x${Cypress.env("NETWORK_ID").toString(16).toUpperCase()}`,
```

Comment:
Please change your IDE line length to be 80 to avoid future unnecessary changes like this in the future

Reply:
Oh, I am having already 80 for IDE line length.